src/renderer/helpers/api/local.js

```diff
@@ -580,6 +580,18 @@ function parseListItem(item) {
         descriptionShort: channel.description_snippet.text
       }
     }
+    case 'HashtagTile': {
+      /** @type {import('youtubei.js').YTNodes.HashtagTile} */
+      const hashtag = item
+
+      return {
+        type: 'hashtag',
+        title: hashtag.hashtag,
```

Comment:
Is this supposed to be an object with property `text` or just a string?
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/27780ee7-1753-4ebe-bb40-98b0c79a4930)

Reply:
Calling `toString()` on it or using the `text` property will work.